apps/teams/views/invitation_views.py

```diff
@@ -19,11 +20,18 @@ def accept_invitation(request, invitation_id: uuid.UUID):
         request.session["invitation_id"] = str(invitation_id)
     else:
         clear_invite_from_session(request)
-    if (
-        request.user.is_authenticated
-        and request.user.email.lower() == invitation.email.lower()
-        and is_member(request.user, invitation.team)
-    ):
+    user_email_matches = False
+    if request.user.is_authenticated:
+        user_email_matches = request.user.email.lower() == invitation.email.lower()
+        if not user_email_matches:
+            # If the current user's email doesn't match the invitation,
+            # check if they have an email that does match.
+            # We don't check verified emails since email verification has the
+            # same level of security as invitations.
+            user_email_matches = EmailAddress.objects.filter(
+                email__iexact=invitation.email, user__id=request.user.id
```

Comment:
How does it happen that a user gets multiple email addresses in the first place?

Reply:
You can change your email address in your user profile and you can also manage emails at `/accounts/email/`